src/crimson/osd/main_config_bootstrap_helpers.cc

```diff
@@ -84,6 +84,58 @@ seastar::future<> populate_config_from_mon()
   });
 }
 
+enum class OptionValue {
+  Uint,
+  Double
+};
+
+// Struct to define a Seastar configuration option
```

Comment:
nit: I'm not sure that this comment provides any addtional information to the reader.
`struct SeastarOption` already means "Struct to define a Seastar option", I'm ok if you would like to keep it though.

Reply:
done